ee/clickhouse/queries/test/test_paths.py

```diff
@@ -58,23 +60,28 @@ class TestClickhousePaths(ClickhouseTestMixin, paths_test_factory(ClickhousePath
 
     maxDiff = None
 
+    def _create_groups(self):
+        GroupTypeMapping.objects.create(team=self.team, group_type="organization", group_type_index=0)
+        GroupTypeMapping.objects.create(team=self.team, group_type="company", group_type_index=1)
+
+        create_group(team_id=self.team.pk, group_type_index=0, group_key="org:5", properties={"industry": "finance"})
+        create_group(team_id=self.team.pk, group_type_index=0, group_key="org:6", properties={"industry": "technology"})
+
+        create_group(
+            team_id=self.team.pk, group_type_index=1, group_key="company:1", properties={"industry": "technology"}
+        )
+        create_group(team_id=self.team.pk, group_type_index=1, group_key="company:2", properties={})
+
     def _get_people_at_path(self, filter, path_start=None, path_end=None, funnel_filter=None, path_dropoff=None):
         person_filter = filter.with_data(
             {"path_start_key": path_start, "path_end_key": path_end, "path_dropoff_key": path_dropoff}
         )
         result = ClickhousePathsPersons(person_filter, self.team, funnel_filter)._exec_query()
         return [row[0] for row in result]
 
+    @test_with_materialized_columns(["$current_url", "$screen_name"], person_properties=["email"])
     def test_denormalized_properties(self):
-        materialize("events", "$current_url")
```

Comment:
Doing this meant no cleanup, since we truncate vs drop tables after tests now. This is problematic for following tests